src/foraging_gui/Dialogs.py

```diff
@@ -1327,8 +1322,7 @@ def _start_preview(self):
             self.MainWindow.Channel.StopCameraPreview(int(1))
 
             self.StartPreview.setStyleSheet("background-color : none;")
-            self.WarningLabelCameraOn.setText('Camera is off')
```

Comment:
Its hard for me to know where each of these warnings was before. Can you add screenshots of before and after for each warning? I know thats a lot of work to document, but its a good way to catch issues

Reply:
Yes will do tomorrow morning!

Reply:
![image](https://github.com/user-attachments/assets/2b06afb2-147b-4787-aaeb-44c8a0b4c35b)

Reply:
I think I would have this label stay where it is, and duplicate it in the queue area. Since its useful to have that information on the camera dialog